Doc/library/typing.rst

```diff
@@ -500,6 +504,8 @@ The module defines the following classes, functions and decorators:
 
     A generic version of :class:`collections.abc.Reversible`.
 
+   .. versionadded:: 3.5.2
```

Comment:
@cblegare Are you sure that ``Reversible`` is new in 3.5.2? I am almost sure it was always there. (It was slightly changed however around a year ago.)